package/Dockerfile.shipyard-linting

```diff
@@ -31,7 +31,7 @@ RUN apk add --no-cache bash findutils git grep make nodejs py3-six shellcheck up
     apk del installers
 
 # Copy shared makefiles so that downstream projects can use it
```

Comment:
nit: probably should update the comment :)

Reply:
I'll generalize it.